hydrocron/api/hydrocron.py

```diff
@@ -2,15 +2,11 @@
 Hydrocron API context module
```

Comment:
I think the connection stuff was the last part of this module that was really doing anything. I think we can remove this module entirely now and just instantiate the DynamoDataRepository directly in the api controllers

Reply:
@frankinspace I have removed the api.hydrocron module and now instantiate the data repository and access in the api.timeseries module.